content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md

```diff
@@ -230,7 +230,7 @@ the kubelet until communication with the API server is re-established. In the me
 the pods that are scheduled for deletion may continue to run on the partitioned node.
 
 {{< note >}}
-The control plane limits the rate of adding node new taints to nodes. This rate limiting
+The control plane limits the rate of adding new node taints to nodes. This rate limiting
```

Comment:
The control plane limits the rate of adding new taints to nodes. This rate limiting